include/ebpf_api.h

```diff
@@ -63,7 +58,7 @@ extern "C"
      * @param[out] map_handles Array of map handles to be filled in.
      * @param[out] error_message Error message describing what failed.
      */
```

Comment:
add `@retval` lines with common return values (but ok to do in a future PR if you prefer, since it was already bad before)

Reply:
I can fix this in the next PR -- I am going to work on implementing the APIs next.